ruff.toml

```diff
@@ -11,9 +11,16 @@ ignore = [
 select = [
     "E",  # pycodestyle errors
     "F",  # pyflakes
+    "I",  # isort
     "W",  # pycodestyle warnings
 ]
 
+[lint.isort]
+combine-as-imports = true
+lines-after-imports = 2
+lines-between-types = 1
+no-sections = true
```

Comment:
I would much prefer not setting `no-sections`, as it doesn't differentiate between local and non-local imports. I don't have a specific section order I prefer (as in, the default would be fine for me), but having _some_ section order would be good.
```suggestion
```

Reply:
> I would much prefer not setting no-sections, as it doesn't differentiate between local and non-local imports.

I'm confused. Removing `no-sections` means that it **will** differentiate (and separate) different type of imports.

For example, `models/projects.py` becomes

```python
from collections import defaultdict
from os.path import basename, join, normpath
from urllib.parse import urlparse

from django.conf import settings
from django.contrib.auth.models import User
from django.db import models
from django.db.models import Prefetch
from django.utils import timezone
from django.utils.functional import cached_property

from pontoon.base import utils
from pontoon.base.models.aggregated_stats import AggregatedStats
from pontoon.base.models.changed_entity_locale import ChangedEntityLocale
from pontoon.base.models.locale import Locale

class…
```

Is that what you want?

Reply:
OK. I'm going to use the section order that I found in docs, let me know if you prefer to drop that.

```
section-order = ["future", "standard-library", "django", "third-party", "first-party", "local-folder"]

[lint.isort.sections]
"django" = ["django"]
```

Reply:
No, that's good.